models/topics-models.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../db/connection')
+
+exports.selectTopics = () => {
+    return db.query(`
+    SELECT * FROM topics;`)
+    .then(({rows}) => {
+        return rows
+    })
+}
```

Comment:
Nice clear function naming in the controller and model
Nice use of destructuring 👍